### ISSUE TITLE
[Bug]: Sign should add image on topmost layer

### ISSUE THREAD

**Issue by Frooodle** [Stirling-Tools/Stirling-PDF#2449]
### Installation Method

None

### The Problem

Sign should add image on topmost layer
currently it can be hidden under other elements

### Version of Stirling-PDF

all

### No Duplicate of the Issue

- [X] I have verified that there are no existing issues raised related to my problem.

**Comment by TowyTowy** [comment]
Traced this against current `main` and I believe the v2 frontend resolved it: the Sign flow now flattens signatures client-side via PDFium, and `embedBitmapImageOnPage` appends the image with `FPDFPage_InsertObject` before `FPDFPage_GenerateContent` — so the signature is the last-painted page-content object and renders on top of all existing page content. The originally reported behavior doesn't reproduce anymore.

One residual case that's technically still true but is a design question rather than a bug: if the source PDF carries *interactive annotations* (form-field widgets, comments), the PDF imaging model always paints annotations above page content, so those would still cover a flattened signature. Beating them would mean either flattening all annotations (destroying form fields) or re-adding the signature as a topmost annotation (undoing the deliberate content-baking in `signatureFlattening.ts`). If that case matters, it probably deserves its own scoped ticket — otherwise this looks closable as fixed by the v2 rewrite.

**Comment by reecebrowne** [comment]
Yes sorry I should have closed this, it has been solved. I think the annotation behaviour you mention is correct as is, a flattened signature should be covered by annotations.
